docs/examples/BasketItem.jsx

```diff
@@ -0,0 +1,39 @@
+var example = (
+  <div className="panel">
```

Comment:
I probably wouldn't add the panel `className` here. It creates unnecessary styling for the docs, and is a bootstrap class. Although the docs uses bootstrap it isn't necessary for the demo.